chainstate/src/detail/transaction_verifier/mod.rs

```diff
@@ -489,6 +531,29 @@ impl<'a, S: BlockchainStorageRead> TransactionVerifier<'a, S> {
         Ok(())
     }
 
+    fn register_tokens_issuance(&mut self, tx: &Transaction) -> Result<(), TokensError> {
```

Comment:
Please add a comment "token registration saves the token id in the database with the transaction that issued it, and possible some additional auxiliary data; this helps in finding the relevant information of the token at any time in the future".

Reply:
Done